### ISSUE TITLE
False positive after lxml 3.4.1 release

### ISSUE THREAD

**Issue by pylint-bot** [PyCQA/pylint#381]
Originally reported by: **Anonymous**

---

Hi guys,

[This changelog](http://lxml.de/3.4/changes-3.4.1.html) doesn't say anything about module restructuration. Yet pylint fails on some imports. I had to do the following:

```
#!python

from lxml.etree import Element  # pylint: disable=E0611
from lxml.etree import CDATA  # pylint: disable=E0611
```

Is this expected? Has it something to do with the fact that lxml relies on underlying libraries?

Thanks!

---
- Bitbucket: https://bitbucket.org/logilab/pylint/issue/381
